edge-modules/ModuleLib/Timers.cs

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft. All rights reserved.
-namespace LoadGen
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil
```

Comment:
Why move this class? btw @dylanbronson may remove this class in PR, https://github.com/Azure/iotedge/pull/2041/files?

Reply:
I was just deleting it. I have restored it in load-gen, along with all other modifications to the module.